after_update.php

```diff
@@ -0,0 +1,22 @@
+<?php
+error_reporting(E_ALL);
+session_start();
+
+
+if ($_POST['nazwa'])
+{
+    require_once 'database/database.php';
+
+    $sql = "UPDATE leki SET Nazwa = ?, Ilosc = ?, Zapotrzebowanie = ? WHERE id = ?";
+    $pdo->prepare($sql)->execute([$_POST['nazwa'], $_POST['ilosc'], $_POST['zapotrzebowanie'], $_SESSION['id']]);
```

Comment:
Po co Ci sesja? do ID dobierasz się przez $_GET['id']

Reply:
Pewnie też dlatego Ci failuje.

Reply:
A jak Ci failuje to kombinuj - wpisuj rzeczy z palca i zobacz, które nie przechodzi.